babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -788,15 +788,70 @@ SqlIndex SqlCreateTableIndex(Span s) :
 SqlNode SqlExecMacro() :
 {
     SqlIdentifier macro;
+    SqlNodeList paramNames = new SqlNodeList(getPos());
+    SqlNodeList paramValues = new SqlNodeList(getPos());
     Span s;
 }
 {
     macro = CompoundIdentifier() { s = span(); }
+    [SqlExecMacroArgument(paramNames, paramValues)]
     {
-        return new SqlExecMacro(s.end(this), macro);
+        return new SqlExecMacro(s.end(this), macro, paramNames, paramValues);
     }
 }
 
+void SqlExecMacroArgument(SqlNodeList paramNames, SqlNodeList paramValues) :
+{
+    SqlNode name;
+    SqlNode value;
+}
+{
+    LOOKAHEAD(2)
```

Comment:
I think the default for babel is a lookahead of 2. Does it work without this statement?

Reply:
Thanks for the information. Removed LOOKAHEAD in https://github.com/googleinterns/calcite/pull/108/commits/1569d122e19bac9c710f4124eb031b942fc9ae12